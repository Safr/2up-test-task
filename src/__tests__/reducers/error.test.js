import { ADD_USERS_FAILURE } from '../../actionTypes';
import errorReducer from '../../reducers/error';

const initialState = '';

describe('Users reducer', () => {
  test('should setup default state', () => {
    const state = errorReducer(undefined, { type: '@@INIT' });
    expect(state).toMatchSnapshot();
  });
  test('should add error', () => {
    const action = { type: ADD_USERS_FAILURE, error: 'error' };
    const state = errorReducer(initialState, action);
    expect(state).toMatchSnapshot();
  });
});
