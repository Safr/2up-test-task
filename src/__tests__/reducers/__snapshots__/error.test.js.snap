// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Users reducer should add error 1`] = `"error"`;

exports[`Users reducer should setup default state 1`] = `""`;
